force_bdss/core_plugins/dummy/ui_notification/ui_notification.py

```diff
@@ -0,0 +1,127 @@
+import logging
+from traits.api import Instance, String
+
+from force_bdss.api import (
+    BaseNotificationListener,
+    MCOStartEvent,
+    MCOFinishEvent,
+    MCOProgressEvent
+)
+
+import zmq
+
+log = logging.getLogger(__name__)
+
+
+class UINotification(BaseNotificationListener):
```

Comment:
Shouldn't this be a plugin coming from the UI and be part of the force_wfmanager repo ?